### PR TITLE
[UPDATE] Start updating content on what a package is

### PR DESCRIPTION
**Pull request by NikRimington**
This is a draft PR for the Packages Team to work on regarding updating details on what a package is.

### CODE REVIEW

**Comment by jmayntzhusen** [comment]
@NikRimington 

Think what is here at the moment is fine! I am a bit on the fence about whether we should include 2 ways of grouping package types as it may lead to confusion. 
Other than that I think all that is needed is more explanation of the other categories 🙂

File: Extending/Packages/index.md [hunk 1]
```diff
@@ -15,6 +15,29 @@ An Umbraco Package can be many things, but are generally characterized by:
...
+ - Starter Kits
+ - Content Apps
+ - Integration Extensions: e.g. E-Commerce packages
+
+ There is also another way of categorising packages which is based around their use-case:
+ 
+ - **Editor Enhancements**: Property Editors, Content Apps, etc. that extends Umbraco with new ways of editing content
+ - **Administration tools**: Dashboards, Health Checks, etc. that helps administrators with a better overview of content
+ - **Starter Kits**: Complete websites or large features that enables both editors and developers to get a head start
+ - **Back office developer functionality** Productivity tools for developers that increases developer productivity and skills
```
Comment (sofietoft) [inline Extending/Packages/index.md]:
```suggestion
 - **Backoffice developer functionality** Productivity tools for developers that increases developer productivity and skills 
```

File: Extending/Packages/index.md [hunk 2]
```diff
@@ -15,6 +15,29 @@ An Umbraco Package can be many things, but are generally characterized by:
 - Solving real life problems
 - Inspiring people on what Umbraco can be made capable of
 
+### Categories of packages
+
+Packages provide a wide variety of functionality, and often they can span multiple categories. In general, though, the functionality they provide fall into these main groups:
```
Comment (sofietoft) [inline Extending/Packages/index.md]:
```suggestion
Packages provide a wide variety of functionality, and can often span multiple categories. In general, though, the functionality they provide fall into these main groups:
```

**Comment by jmayntzhusen** [comment]
@sofietoft this is now ready for review 🙂

**Comment by dampee** [comment]
Pretty sure @sofietoft overlooked your remark :-D

File: Extending/Packages/index.md [hunk 3]
```diff
@@ -19,6 +19,41 @@ An Umbraco Package can be many things, but are generally characterized by:
...
+
+ - [Schema Extensions](#schema-extensions)
+ - [Management Extensions](#management-extensions)
+ - [Starter Kits](#starter-kits)
+ - [Content Apps](#content-apps)
+ - [Integration Extensions](#integration-extensions)
+
+#### Schema Extensions
+
+A package that can be categorised as a Schema Extension will extend the default Umbraco Schema. Schema in this sense is things like Data Types / Property Editors / Document Types / Media Types. By extending Umbraco with packages such as [Our.Umbraco.GMaps](https://our.umbraco.com/packages/backoffice-extensions/ourumbracogmaps-google-maps-for-umbraco-8/) it gives editors greater capabilities when they are populating their content pages.
```
Comment (sofietoft) [inline Extending/Packages/index.md:34]:
```suggestion
A package that can be categorised as a Schema Extension will extend the default Umbraco Schema. Schema in this sense is things like Data Types, Property Editors, Document Types and Media Types. By extending Umbraco with packages such as [Our.Umbraco.GMaps](https://our.umbraco.com/packages/backoffice-extensions/ourumbracogmaps-google-maps-for-umbraco-8/) it gives editors greater capabilities when they are populating their content pages.
```

File: Extending/Packages/index.md [hunk 4]
```diff
@@ -19,6 +19,41 @@ An Umbraco Package can be many things, but are generally characterized by:
...
+
+#### Management Extensions
+
+A Management Extension package helps you manage your site, and provides info to the users. Management extensions can be made of many different parts, but often they include custom sections or dashboards for you to manage something.
+A good example of a management extension package is [Diplo God Mode](https://our.umbraco.com/packages/developer-tools/diplo-god-mode/) which gives you a lot of extra info and tools to manage your site.
+
+#### Starter Kits
+
+Starter kits are, as the name suggests, a package that helps you set up a starter version of whatever you want to build. Most starter kit packages are for starting a website, so they include doc types, templates, data types, content nodes and media. There are also some quite specialized starter kits, for example for creating a blog. Umbraco HQ has released their [own starter kit](https://our.umbraco.com/packages/starter-kits/the-starter-kit/), that creates a small site with the most commonly used parts.
```
Comment (sofietoft) [inline Extending/Packages/index.md:43]:
```suggestion
Starter kits are, as the name suggests, a package that helps you set up a starter version of whatever you want to build. Most starter kit packages are for starting a website, and include schema like Document Types and Templates as well as content nodes and media. There are also some quite specialized starter kits, for example for creating a blog. Umbraco HQ has released their [own starter kit](https://our.umbraco.com/packages/starter-kits/the-starter-kit/), that creates a small site with the most commonly used parts. 
```

File: Extending/Packages/index.md [hunk 5]
```diff
@@ -19,6 +19,41 @@ An Umbraco Package can be many things, but are generally characterized by:
...
+
+:::note
+Content Apps are only available from Umbraco 8
+:::
+
+Content apps are almost like dashboards for content nodes that are intended to display node specific information. A good example of this is the [Preflight](https://our.umbraco.com/packages/backoffice-extensions/preflight-content-health-checks-for-umbraco-8/) content app. It shows you readability scores for your written content, directly on each content node.
+
+#### Integration extensions
+
+This type of package can be a lot of things, and can include a lot of the other package types in it. They are generally integrating a larger system into Umbraco, a good example could be an ecommerce package such as [UCommerce](https://our.umbraco.com/packages/website-utilities/ucommerce/), that includes an entire webshop module.
```
Comment (sofietoft) [inline Extending/Packages/index.md:55]:
```suggestion
This type of package can be a lot of things, and can include a lot of the other package types in it. They are generally integrating a larger system into Umbraco. A good example could be an e-commerce package such as [UCommerce](https://our.umbraco.com/packages/website-utilities/ucommerce/), that includes an entire webshop module for Umbraco.
```

**Pull request merged**